apps/music/views/albums/view.js

```diff
@@ -64,31 +60,19 @@ AlbumsView.prototype.render = function() {
 };
 
 AlbumsView.prototype.getAlbums = function() {
-  return document.l10n.formatValues('unknownAlbum', 'unknown')
-    .then(([unknownAlbum, unknown]) => {
+  return document.l10n.formatValues('unknownAlbum', 'unknownArtist')
```

Comment:
see below on line 73.. we show the `artist` as the sub-text in the list items and we weren't falling back to the 'unknownArtist' string if that field was blank